packages/fxa-payments-server/src/en.ftl

```diff
@@ -0,0 +1,81 @@
+## App-level string(s) and messages shared by multiple components
+
+document =
+  .title = Firefox Accounts
+
+# General aria-label for closing modals
+close-aria =
+  .aria-label = Close modal
+
+# AppLayout Component and Subscriptions Route
+settings-subscriptions-title = Subscriptions
+
+# UNUSED STRINGS?
```

Comment:
Strings from line 13 onwards don't seem to be used in any routes/components. Are these safe to delete?

Reply:
👍 Confirmed that they are all legacy strings and can be removed.